src/main/java/uk/gov/hmcts/darts/transcriptions/validator/TranscriptionApproveMarkForDeletionValidator.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.darts.transcriptions.validator;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.authorisation.component.UserIdentity;
+import uk.gov.hmcts.darts.common.component.validation.Validator;
+import uk.gov.hmcts.darts.common.entity.ObjectHiddenReasonEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.common.exception.DartsApiException;
+import uk.gov.hmcts.darts.common.repository.ObjectAdminActionRepository;
+import uk.gov.hmcts.darts.common.repository.ObjectHiddenReasonRepository;
+import uk.gov.hmcts.darts.transcriptions.exception.TranscriptionApiError;
+
+import static java.util.Objects.nonNull;
+
+@Component
+@RequiredArgsConstructor
+public class TranscriptionApproveMarkForDeletionValidator implements Validator<Integer> {
+
+    private final ObjectAdminActionRepository objectAdminActionRepository;
+    private final ObjectHiddenReasonRepository objectHiddenReasonRepository;
+    private final UserIdentity userIdentity;
+    private final TranscriptionDocumentIdValidator transcriptionDocumentIdValidator;
+
+    @Override
+    public void validate(Integer transcriptionDocumentId) {
+        transcriptionDocumentIdValidator.validate(transcriptionDocumentId);
+
+        var objectAdminActionEntityList = objectAdminActionRepository.findByTranscriptionDocument_Id(transcriptionDocumentId);
+        if (objectAdminActionEntityList.isEmpty()) {
+            throw new DartsApiException(TranscriptionApiError.TRANSCRIPTION_DOCUMENT_ID_NOT_FOUND);
+        } else if (objectAdminActionEntityList.size() > 1) {
```

Comment:
Else if not needed can just use if

Reply:
else stops the evaluation of extra condition here. anyways changes as you requested.